.github/workflows/run-tests.yml

```
@@ -26,7 +26,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Parse Node version
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        run: echo " NVMRC=$(cat .nvmrc)" >> $GITHUB_OUTPUT
```

I think an extra space has crept in here:

```suggestion
        run: echo "NVMRC=$(cat .nvmrc)" >> $GITHUB_OUTPUT
```